README.md

```diff
@@ -1 +1,333 @@
+- Twitter: [@tchasinga](https://twitter.com/tchasinga)
+- LinkedIn: [tchasinga](https://linkedin.com/in/tchasinga)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> here's any best  1 - 3 features  that I will add to the project.
+
+- [ ] **[Full-management-designer]**
+- [ ] **[UI-animating]**
+- [ ] **[Wecpack]**
+
+<p align="right">(<a href="#readme-top">return to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🚀 Live Demo <a name='demo'></a>
+> Right now this website is Pending , Hope you come back later thanks you []() 
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+>Thank you for inviting me to contribute to your project! I'd be happy to share some ideas and suggestions. Here are a few additional considerations to enhance your project:
+
+1. **Code splitting and lazy loading**: Implement code splitting and lazy loading techniques to improve the performance of your application. This allows you to load only the necessary code for each page or component, reducing the initial bundle size and improving the load time.
+
+2. **Environment-specific configurations**: Consider creating different configuration files for different environments (e.g., development, production) to tailor the build process and optimizations accordingly. This can include setting different values for variables, enabling specific plugins, or applying different optimizations based on the deployment environment.
+
+3. **Asset optimization**: Explore additional optimization techniques for assets such as images, fonts, and other media files. You can utilize loaders and plugins specific to these file types to optimize their size and format, improving the overall performance of your application.
+
+4. **Source mapping**: Enable source mapping for your JavaScript and CSS files. Source maps allow for easier debugging and help in identifying issues by mapping the minified/optimized code back to the original source code.
+
+5. **Cache busting**: Implement cache-busting techniques to ensure that users receive the latest version of your bundled files when you make updates. This can be achieved by appending a unique hash to the filenames, forcing browsers to fetch the updated files instead of using cached versions.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+>If you like this project, please think about doing anything you can to support us! Spreading the word about our mission, volunteering your time and talents, or even just making a small donation all assist us to continue making a positive difference. With your assistance, we can transform the world and create a better future for everyone. I value your help and consideration.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] Your acknowledgement should give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :+1: